### ISSUE TITLE
When sidebar is open, grayscaling leaves part of header white

### ISSUE THREAD

**Issue by ardacebi** [Decury/decury#27]
When the sidebar is expanded, a part of the header stays white rather than get grayscaled.

Just expand the sidebar and see what I'm talking about.

<img width="1473" alt="Screen Shot 2020-03-26 at 21 10 39" src="https://user-images.githubusercontent.com/17576065/77681364-46b8ec80-6fa6-11ea-8910-e5e9c080a31f.png">

**Comment by Yutyo** [comment]
@ardacebi Nope, I think its good that way. If anyone has any idea, can let us know.
